main.go

```diff
@@ -49,6 +49,7 @@ var (
 	concurrency         = cli.Flag("concurrency", "Number of concurrent workers.").Default(strconv.Itoa(runtime.NumCPU())).Int()
 	noVerification      = cli.Flag("no-verification", "Don't verify the results.").Bool()
 	onlyVerified        = cli.Flag("only-verified", "Only output verified results.").Bool()
+	forceReverification = cli.Flag("allow-verification-overlap", "Allow verification of similar credentials across detectors").Bool()
```

Comment:
Please update this flag naming

Reply:
Should we rename all the `reverification` elements? `reverificationWorkers`, etc